noxfile.py

```diff
@@ -1,7 +1,7 @@
 from nox_poetry import Session, session
 
 
-@session(python=["3.7", "3.8", "3.9", "3.10"])
+@session(python=["3.8", "3.9", "3.10"])
```

Comment:
Nice. We should also look at the GitHub Actions in case we need to update it to drop py37.

Reply:
Working on that right now! That would mean modifying the test-release.yml file, right? That's what I've done right now.

Reply:
Correct.